s4e-backend/src/main/java/pl/cyfronet/s4e/service/UserRoleService.java

```diff
@@ -21,15 +23,19 @@
     private final AppUserRepository appUserRepository;
     private final GroupRepository groupRepository;
 
+    @Transactional
+    public void addRole(CreateUserRoleRequest request) throws NotFoundException {
+        addRole(request.getRole(), request.getEmail(), request.getInstitutionSlug(), request.getGroupSlug());
+    }
+
     @Transactional
```

Comment:
Don't you need to specify the `NotFoundException` here? (`rollbackFor`, or sth like this)

Reply:
why should i?